app/Http/Controllers/BookshelfController.php

```diff
@@ -101,6 +101,9 @@ public function show(string $slug)
         $shelf = $this->bookshelfRepo->getBySlug($slug);
         $this->checkOwnablePermission('book-view', $shelf);
 
+        $sort = setting()->getForCurrentUser('books_sort', 'name');
+        $order = setting()->getForCurrentUser('books_sort_order', 'asc');
```

Comment:
Within `show.blade.php` changes I've added a comment regarding user-defined ordering. Once that's added we won't be able to share the `books_sort` setting with the books listing since there would be differences in available ordering options. We'd probably need to add a `shelf_books_sort` option, which defaults to the current user-defined sort.